modules/user_auth/files/kratos-values.yml

```diff
@@ -0,0 +1,156 @@
+image:
+  repository:  oryd/kratos
+  tag: v0.5.4-alpha.1-sqlite
+  pullPolicy: IfNotPresent
+
+ingress:
+  public:
+    enabled: true
+    annotations:
+        kubernetes.io/ingress.class: nginx
+        # nginx.ingress.kubernetes.io/rewrite-target: /$1
+        cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
+    hosts: # Filled by user_auth.tf
+      - paths: ["/"]
+    tls:
+      - secretName: kratos-public-tls-secret
+
+  admin:
+    enabled: true
+    annotations:
+        kubernetes.io/ingress.class: nginx
+        # nginx.ingress.kubernetes.io/rewrite-target: /$1
+        cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
+    hosts: # Filled by user_auth.tf
+      - paths: ["/"]
+    tls:
+      - secretName: kratos-admin-tls-secret
+
+# To set up OIDC you can uncomment this and create a secret which contains the
+# OIDC provider configuration in JSON format in an env var called SELFSERVICE_METHODS_OIDC
+# See README.md for more info
+# deployment:
+  # environmentSecretsName: secretName
+
+kratos:
+  development: false
+  autoMigrate: true
+
+#  You can add multiple identity schemas here
+  identitySchemas:
+    "identity.default.schema.json": |
+      {
+        "$id": "default",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "title": "Person",
+        "type": "object",
+        "properties": {
+          "traits": {
+            "type": "object",
+            "properties": {
+              "email": {
+                "type": "string",
+                "format": "email",
+                "ory.sh/kratos": {
+                  "credentials": {
+                    "password": {
+                      "identifier": true
+                    }
+                  },
+                  "recovery": {
+                    "via": "email"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+  config:
+    secrets:
+      default: ["test-insecure-session-id-here"]
```

Comment:
Hmm just noticed this never got filled in..

Reply:
does this not get override by `set` in user_auth.tf ?
```
  # This secret contains db credentials created during the initial zero apply command
  set {
    name  = "secret.nameOverride"
    value = var.project
  }
```

Reply:
No, this one is used as a secret key for signing cookies. Maybe we can just use the project seed and name for now or something. Mind making this a variable for the module?

Reply:
You can use `sensitive   = true` in the definition